modules/gapi/include/opencv2/gapi/gmat.hpp

```diff
@@ -133,13 +133,16 @@ static inline GMatDesc empty_gmat_desc() { return GMatDesc{-1,-1,{-1,-1}}; }
 class Mat;
 GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat);
 GAPI_EXPORTS GMatDesc descr_of(const cv::UMat &mat);
+GAPI_EXPORTS std::vector<GMatDesc> descr_of(std::vector<cv::Mat> const& vec);
+GAPI_EXPORTS std::vector<GMatDesc> descr_of(std::vector<cv::UMat> const& vec);
 #endif // !defined(GAPI_STANDALONE)
 
 /** @} */
 
 namespace gapi { namespace own {
     class Mat;
     GAPI_EXPORTS GMatDesc descr_of(const Mat &mat);
+    GAPI_EXPORTS std::vector<GMatDesc> descr_of(std::vector<Mat> const& vec);
```

Comment:
Did you check with Anton if overriding-in-a-namespace is ok for this purpose? W.r.t ADL and so on